docs/source/model_design/vivarium_model_components/GBD_disease_health/index.rst

```diff
@@ -79,7 +79,9 @@ not modeled with the same complexity as causes.
 Sequelae detail a specific medical condition associated with 
```

Comment:
Not really related to this PR, but could we add "Sequelae are only used in GBD non-fatal estimates; deaths are attributed to a cause but not to a sequela within that cause." to the previous paragraph? Assuming that sounds correct 😄